lib/shared/store.js

```diff
@@ -229,6 +231,8 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
     }
 
     case ACTIONS.SET_FLASH_STATE: {
+      // action.data : Object
```

Comment:
Nice, but given there's now so many lines that say `// action.data : Object`, I wonder if these too should be a bit more verbose e.g. `// action.data : Object (flash state)` ? (or `// action.data : flash state Object`)
Or perhaps that's unnecessary as that info is implicit from the `case` line (e.g. `ACTIONS.SET_FLASH_STATE`) ?

Reply:
Done that now (and also the linter requires sentence-case in comments, so that's fixed too)